server/controllers/messageController.js

```diff
@@ -2,14 +2,14 @@ const Message = require("../models/Message");
 
 exports.create = async (req, res, next) => {
   try {
-    const message = new Message({
+    const newMessage = new Message({
       author: req.user._id,
       content: req.body.content,
       conversation: req.params.convoID,
```

Comment:
if this is params, we don't need to add it to the body of the content we send

Reply:
Im not quite sure what you mean by this?

Reply:
On the front end, you include conversationId in the body of the POST request, but here you're reading if from the path parameters instead of the body.